model-view-intent/src/main/java/com/iluwatar/model/view/intent/App.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * This project is licensed under the MIT license.
+ * Module model-view-viewmodel is using ZK framework licensed under LGPL
+ * (see lgpl-3.0.txt).
+ *
+ * The MIT License
+ * Copyright © 2014-2022 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.iluwatar.model.view.intent;
+
+/**
+ * Model-View-Intent is a pattern for implementing user interfaces.
+ */
+public final class App {
+
+
+  /**
+   * To avoid magic value lint error.
+   */
+  private static final double RANDOM_VARIABLE = 10.0;
+
+  /**
+   * Program entry point.
+   *
+   * @param args command line args
+   */
+  public static void main(final String[] args) {
+    // create model, view and controller
+    CalculatorView view = new CalculatorView();
+    double variable1 = RANDOM_VARIABLE;
+    view.setVariable(variable1);
+    view.add();
+    view.displayTotal();
+
+    variable1 = 2.0;
+    view.setVariable(variable1);
+    view.subtract();
+    view.divide();
+    view.multiply();
+    view.displayTotal();
```

Comment:
- Use `var` for local variables, where possible
- Add more comments for the audience (this is study material)